routes/api.js

```diff
@@ -39,6 +39,15 @@ api.get('/user/:secret', function (req, res) {
     })
 })
 
+// Route for getting exposure count for user
+api.get('/user/:secret/exposure', async function (req, res) {
```

Comment:
I thought the route was `/api${this.props.location.pathname}/exposure`

Reply:
> start.js
<img width="582" alt="Screen Shot 2020-07-22 at 12 38 05 PM" src="https://user-images.githubusercontent.com/18102685/88203681-304f3480-cc18-11ea-95b2-8442c7775303.png">

When the request is forwarded to the API router, the `/api` part is chopped off. Or to think of it the other way around, the route definition has `/api` pre-pended to it.